.travis.yml

```diff
@@ -7,14 +7,11 @@ node_js:
   - "6"
   - "7"
 
-matrix:
-  allow_failures:
-    - node_js: "0.12"
-    - node_js: "node"
- 
 branches:
-  except:
-    - canary
+  only:
+    - master
+    - beta
+    - release
```

Comment:
I'm unsure about homu and greenkeeper... I'll monitor and confirm that today. I was mostly annoyed that I was backed up on test runs and trying to get out of that mess.